src/mlx/warnings/robot_checker.py

```diff
@@ -83,11 +84,14 @@ def return_check_limits(self):
         '''
         count = 0
         for checker in self.checkers:
-            if checker.name:
-                print('Counted failures for test suite {!r}.'.format(checker.name))
+            if checker.suite_name:
+                string = f"test suite {checker.suite_name!r}"
+                padded_string = f"{string:<30}"
+                count += checker.return_check_limits(padded_string)
             else:
-                print('Counted failures for all test suites.')
-            count += checker.return_check_limits()
+                string = "all test suites"
+                count += checker.return_check_limits(f"{string:<30}")
+        print(f"Returning error code {self.count}.")
```

Comment:
If the exit code of the tool is 0, it is not an error code. 

```suggestion
        if count:
            print(f"Returning error code {count}.")
```